_query-dsl/specialized/more-like-this.md

```diff
@@ -0,0 +1,282 @@
+        "_source": {
+          "name": "Diana",
+          "alias": "Wonder Woman",
+          "quote": "Justice will come when it is deserved."
+        }
+      },
+      {
+        "_index": "articles-optimized",
+        "_id": "a2",
+        "_score": 1.1596459,
+        "_source": {
+          "name": "Clark",
+          "alias": "Superman",
+          "quote": "Even in the darkest times, hope cuts through."
+        }
+      }
+    ]
+  }
+}
+```
+
+Use this pattern when you want to boost results based on a new concept that is not yet fully indexed, but also want to combine it with knowledge from existing indexed documents.
+{: .note}
+
+# Parameters
+
+The only required parameter for a `more_like_this` query is `like`. The rest of the parameters have default values but allow fine-tuning. Parameters fall into three main categories:
+
+## Document Input Parameters
+
+| Parameter | Required/Optional | Data Type | Description |
+| `like`| Required| Array of strings or objects | Defines the text or documents to find similar documents for. You can input free text, real documents from the index, or even artificial documents. The analyzer associated with the field will process the text unless you override it. |
+| `unlike`| Optional| Array of strings or objects | Provides text or documents whose terms should be *excluded* from influencing the query. Useful for specifying negative examples.|
+| `fields`| Optional| Array of strings| Lists fields to use when analyzing text. If not specified, all fields are used. |
+
+## Term Selection Parameters
+
+| Parameter | Required/Optional | Data Type| Description|
+| `max_query_terms` | Optional| Integer| Sets the maximum number of terms to select from the input. A higher value increases precision but slows down execution. Default is `25`. |
+| `min_term_freq` | Optional| Integer| Terms appearing fewer times than this in the input will be ignored. Default is `2`.|
+| `min_doc_freq`| Optional| Integer| Terms that appear in fewer documents than this value will be ignored. Default is `5`.|
+| `max_doc_freq`| Optional| Integer| Terms appearing in more documents than this limit are ignored. Useful for avoiding very common words. Default is unlimited (`2147483647`). |
+| `min_word_length` | Optional| Integer| Ignore words shorter than this value. Default is `0`.|
+| `max_word_length` | Optional| Integer| Ignore words longer than this value. Default is unlimited. |
+| `stop_words`| Optional| Array of strings | Defines a list of words that are ignored completely when selecting terms.|
+| `analyzer`| Optional| String | The custom analyzer to use for processing input text. Defaults to the analyzer of the first field listed in `fields`.|
+
+## Query Formation Parameters
+
+| Parameter | Required/Optional | Data Type| Description|
+| `max_query_terms` | Optional| Integer| Sets the maximum number of terms to select from the input. A higher value increases precision but slows down execution. Default is `25`. |
```

Comment:
This is a duplicate list from the previous table. It should contain query formation parameters `minimum_should_match`, `fail_on_unsupported_field`, `boost_terms`, `include`, and `boost`.

Reply:
Absolutely, was there as a place holder and forgot to update it, thats done now